README.md

```diff
@@ -1,27 +1,41 @@
 Cloudinary Android SDK
 ======================
+[![Build Status](https://api.travis-ci.com/cloudinary/cloudinary-android.svg?branch=master)](https://app.travis-ci.com/github/cloudinary/cloudinary_android)
 
-Cloudinary is a cloud service that offers a solution to a web application's entire image management pipeline.
+## About
+The Cloudinary Android SDK allows you to quickly and easily integrate your application with Cloudinary.
+Effortlessly optimize and transform your cloud's assets.
 
-Easily upload images to the cloud. Automatically perform smart image resizing, cropping and conversion without installing any complex software.
-Integrate Facebook or Twitter profile image extraction in a snap, in any dimension and style to match your website’s graphics requirements.
-Images are seamlessly delivered through a fast CDN, and much much more.
+### Additional documentation
+This Readme provides basic installation and usage information.
+For the complete documentation, see the [Android SDK Guide](https://cloudinary.com/documentation/android_integration).
 
-Cloudinary offers comprehensive APIs and administration capabilities and is easy to integrate with any web application, existing or new.
+## Table of Contents
+- [Key Features](#key-features)
+- [Compatibility](#Compatibility)
+- [Installation](#installation)
+- [Usage](#usage)
+    - [Setup](#Setup)
+    - [Transform and Optimize Assets](#Transform-and-Optimize-Assets)
+    - [Uploading Asset](#Uploading)
 
-Cloudinary provides URL and HTTP based APIs that can be easily integrated with any Web development framework.
+## Key Features
+- [Image Transformation](https://cloudinary.com/documentation/android_image_manipulation) and [Video Transformation](https://cloudinary.com/documentation/android_video_manipulation) assets (links to docs).
 
-## Requirements
-The library requires Android version 4.0.3 (Ice Cream Sandwich) and up.
+## Compatibility
+* Minimum Android SDK: Cloudinary requires a minimum API level of 19.
+* Compile Android SDK: Cloudinary requires you to compile against API 31 or later.
```

Comment:
I have some wording suggestions, but I don't really understand what it means that we "require" level 19, but that you have to compile against 31?  What is it that you can do with level 19 after you've compiled against 31? Is 19 the level of Android that the end-users of the mobile app need to have?

Reply:
My comment here is two fold:

1. We should use a table to make it clearer (as the template shows)
2. Required/Compile might confuse people, it might be very common for Android developers to use this terminology, but it still looks confusing.

Reply:
After talking to @patrick-tolosa we are removing the compile SDK version as it might confuse people 
After looking at the top SDK's for Android on git I'll only put the minimum SDK version required for our SDK
As its name implies the minimum SDK version means that lower version cannot compile/work with our SDK.

Reply:
This is another suggestion from another Android repo.
Cloudinary requires at minimum Java 8+ or Android API 19+.

Reply:
This is another suggestion from another repo
Cloudinary requires at minimum Java 8+ and Android API 19+.

Reply:
After talking with @patrick-tolosa we decided to use a table to be consistent with other SDK's

| Cloudinary SDK | Android SDK | 
|----------------|-------------|
|      1.x       |    > 14     |
|      2.x       |    > 19     |